gen-ai/inference/vllm-rayserve-gpu/ray-service-vllm.yaml

```diff
@@ -131,9 +138,11 @@ spec:
               value: "8000"
             - name: LD_LIBRARY_PATH
               value: "/home/ray/anaconda3/lib:$LD_LIBRARY_PATH"
-          nodeSelector:
-            NodeGroupType: g5-gpu-karpenter
-            type: karpenter
```

Comment:
I think we need this to use the Karpenter Nodepools to spin up the nodes

Reply:
Oversight on my part, I do this differently and meant to leave this in. Let me revert